.github/workflows/ci.yml

```diff
@@ -0,0 +1,155 @@
+      HELM_CHART: ${{ matrix.chart }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Installing kubeval
+        run: helm plugin install https://github.com/instrumenta/helm-kubeval
+      - name: Lint helm chart 
+        run: make helm-lint
+      - name: Validate helm chart 
+        run: make helm-validate
+    strategy:
+      matrix: ${{ fromJson(needs.detect-repo-changes.outputs.helms-changed) }}
+
+ 
+  # test-upgrade:
+  #   name: Upgrade Helm Chart '${{ matrix.chart }}'
+  #   if: ${{ needs.detect-repo-changes.outputs.no-changes != 'true' }}
+  #   needs: [check, detect-repo-changes, dependencies]
+  #   runs-on: ubuntu-20.04
+  #   env:
+  #     HELM_CHART: ${{ matrix.chart }}
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v2
+  #     - name: Install k3d 
+  #       run: |
+  #         export PATH=".bin:$PATH"
+  #         curl -s https://raw.githubusercontent.com/k3d-io/k3d/v5.4.4/install.sh | USE_SUDO="false" K3D_INSTALL_DIR=".bin" bash
+  #     - name: Test upgrade for helm chart 
+  #       run: make helm-upgrade
+  #   strategy:
+  #     matrix: ${{ fromJson(needs.detect-repo-changes.outputs.helms-changed) }}
+ 
+  test-install:
+    name: Install Helm Chart '${{ matrix.chart }}'
+    if: ${{ needs.detect-repo-changes.outputs.no-changes != 'true' }}
+    needs: [check, detect-repo-changes, dependencies]
+    runs-on: ubuntu-20.04
+    env:
+      HELM_CHART: ${{ matrix.chart }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install k3d 
+        run: |
+          export PATH=".bin:$PATH"
+          curl -s https://raw.githubusercontent.com/k3d-io/k3d/v5.4.4/install.sh | USE_SUDO="false" K3D_INSTALL_DIR=".bin" bash
+      - name: Test install for helm chart 
+        run: make helm-test
+    strategy:
+      matrix: ${{ fromJson(needs.detect-repo-changes.outputs.helms-changed) }}
+
+  release:
+    name: Release
+    if: ${{ github.ref_type == 'tag' }}
+    # needs: [test-install, test-upgrade]
+    needs: [test-install]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Define current tag version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Generate documentation 
+        run: |
+            pushd .bin
+            curl -L https://github.com/norwoodj/helm-docs/releases/download/v1.5.0/helm-docs_1.5.0_Linux_x86_64.tar.gz -o helm-docs.tar.gz
+            tar -xvf helm-docs.tar.gz
+            ./helm-docs -c ../helm/charts
+            popd
+      - name: Release helm charts
+        run: make release
+      - name: Push commit for release 
+        run: |
+          git config --global user.email "aeneas@ory.sh"
+          git config --global user.name "aeneasr"
+          git checkout -b make-release HEAD
+          git add -A 
+          git commit -a -m "Release ${RELEASE_VERSION}
+          
+          [skip ci]"
```

Comment:
[`[skip-ci]`](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/) prevent having a loop of CI runs :D